curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/65b2f6acce65b7a69751a090.md

```diff
@@ -0,0 +1,343 @@
+---
+id: 65b2f6acce65b7a69751a090
+title: Step 139
+challengeType: 0
+dashedName: step-139
+---
+
+# --description--
+
+In the `update` function, change `text.innerText` to `text.innerHTML` so that the HTML emote codes 
```

Comment:
Since this will be the first time that `innerHTML` is introduced, we should add more to the description here and include a code example.

Something along these lines 

````md
In order for the `&#x2620;` emoticon text to properly display on the page, you will need to use the <dfn>innerHTML</dfn> property. 

The `innerHTML` property allows you to access or modify the content inside an HTML element using JavaScript. 
 
Here is an example of updating the content for this paragraph element using the `innerHTML`  property.

```html
  <p id="demo">This is a paragraph.</p>
```
```js
    document.querySelector("#demo").innerHTML = "Hello, innerHTML!";
```
 
In the `update` function, change `text.innerText` to `text.innerHTML`. 
````

Reply:
Right thank you. That does make a lot of a sense. And might work pretty well.

Reply:
![Preview](https://github.com/freeCodeCamp/freeCodeCamp/assets/9373245/46112f16-61c3-4a63-a1b2-838297d63436)

What do you think?

Reply:
looks good 👍

Reply:
@a2937 

It doesn't look like you pushed up a commit to add the new changes

Reply:
Oops. Pushing it now.